Exercises/strings/strings.js

```diff
@@ -0,0 +1,28 @@
+// Task #1
+const a = "\u2764";
+console.log(a);
+
+// Task #2
+const b = "This book costs 100\$";
+console.log(b);
+
+// Task #3
+const c = 'Cat';
+
+// Task #4
+console.log(`My favorite animal is ${c}`);
+
+// Task #5
+function CapitalizeFirstLetters(text) {
+    let temp = '';
+    let res = '';
+    for (let i = 0; i < text.length; i++) {
+        temp += text[i];
+        if (text[i] == ' ' || i == text.length - 1) {
```

Comment:
wow

Reply:
you can simplify using `split` but gj

Reply:
👍

Reply:
btw function names should not start with capital letters